src/main.cpp

```diff
@@ -1220,10 +1220,10 @@ bool AcceptToMemoryPoolWorker(CTxMemPool& pool, CValidationState &state, const C
 }
 
 bool AcceptToMemoryPool(CTxMemPool& pool, CValidationState &state, const CTransaction &tx, bool fLimitFree,
-                        bool* pfMissingInputs, bool fOverrideMempoolLimit, bool fRejectAbsurdFee)
+                        bool* pfMissingInputs, bool fOverrideMempoolLimit, const CAmount nAbsurdFee)
```

Comment:
Having pass-by-value args const is something we don't do anywhere, whereas there are many places where CAmount is passed by reference (as a `git grep` shows me). The idea is that CAmount is not a basic type in some derived code.